jni-gen/src/generators/class.rs

```diff
@@ -69,6 +69,7 @@ impl<'a> ClassGenerator<'a> {
             "use jni::signature::*;",
             "use once_cell::sync::OnceCell;",
             "use std::str::FromStr;",
+            "use crate::wrappers::builtins::*;",
```

Comment:
```suggestion
            "use super::builtins::*;",
```
When builting the `modules_tree` in `generate_module`, you can import `builtins` from each parent module:
```rust
res.push(format!("pub mod {name} {{\n"));
```
:arrow_down: 
```rust
res.push(format!("pub mod {name} {{\n"));
res.push(format!("use super::builtins;\n"));
```

Reply:
Thanks, I could not figure this out.